examples/resources/nios_dtc_pool/resource.tf

```
@@ -56,9 +55,9 @@ resource "nios_dtc_pool" "dtc_pool2" {
 
 //create a DTC pool with consolidated monitors 
 //Steps:
-//-Create a DTC pool without consolidated monitors 
-//- Assoiciate the DTC pool with a DTC LBDN that has a zone with infoblox.localdomain member 
-//- Update this DTC pool to add consolidated monitors 
+//Create a DTC pool without consolidated monitors 
```

```suggestion
// Create a DTC pool without consolidated monitors 
```
Nitpick : Please check this for other comments too